### DISCUSSION TITLE
Documentation for migrating BitBucket Cloud to GitHub Enterprise Cloud

### DISCUSSION THREAD

**Discussion by Ant0wan** [community/community#69899] [Enterprise]
### Select Topic Area

Question

### Body

Hi,

We are a new company trying GitHub Enterprise Cloud. We are wondering whether there are tools available to migrate from Bitbucket Cloud to GitHub Enterprise Cloud.

We found the `gh bbs2gh migrate-repo` tool and followed the documentation [here](https://docs.github.com/en/enterprise-cloud@latest/migrations/using-github-enterprise-importer/understanding-github-enterprise-importer/migrating-from-bitbucket-server-with-github-enterprise-importer). However, we encountered errors during the execution of `gh bbs2gh migrate-repo`.

As a result, we've raised the following questions for the GitHub Support Team:

- Can this tool migrate from Bitbucket Cloud to GitHub Enterprise Cloud?
- Does this tool migrate all webhooks related to CI, Jira, and so on?
- Does this tool migrate all open PRs, comments, and so on?
Initially, we also wanted to use GitHub Actions. Additional support to perform such a migration would be greatly appreciated. We still have a few days before deciding if and where to migrate.

Best regards,

**Comment by ghostinhershell** [comment]
Hi @Ant0wan 👋🏾 

Thanks so much for reaching out in the community. At this time Bitbucket Cloud is not included in the GitHub Enterprise Importer offerings. We do have some recommendations for migrating data stored in Bitbucket Cloud. By utilizing our process for [Importing an external Git repository using the command line](https://docs.github.com/migrations/importing-source-code/using-the-command-line-to-import-source-code/importing-an-external-git-repository-using-the-command-line#title-h1), you should be able to migrate data stored in Bitbucket Cloud to a your GitHub repository.

If you don't want to import utilizing the command line, we also have some [documentation](https://confluence.atlassian.com/confeval/development-tools-evaluator-resources/bitbucket/bitbucket-migrate-bitbucket-cloud-to-bitbucket-server) which details a process for migrating the Bitbucket data into a Bitbucket Server first, which should allow you to then migrate from Server into a GitHub repository utilizing the Enterprise Importer. **It should also be noted that this particular process is outside of our support offerings and would be utilized at your own risk.**

As for the addition of Bitbucket Cloud to the GEI, we can't say if or when we may add a feature, but I recommend you keep an eye on the [GitHub Blog](https://github.com/blog) and our social media feeds for the latest announcements about new features and fixes. You can also browse our [Public roadmap](https://github.com/orgs/github/projects/4247/views/1) which is a project board that provides a high level overview of the current developments and their planned release schedule

Let me know if you have any additional questions. Thanks!

**Reply by Ant0wan** [reply]
Thanks for the answer.
Unfortunately I won't process the migration. I am no longer working for this company. I wanted to thank you for your help. 
Best Regards,

**Reply by ghostinhershell** [reply]
No problem, best regards to you as well!

**Reply by playtowork** [reply]
I was trying to use the bbs2gh tool as well, until I realized it wasn't working because it expected a Bitbucket Server source (whereas I have a Bitbucket Cloud source). So I might try [importing an external Git repository using the command line](https://docs.github.com/en/migrations/importing-source-code/using-the-command-line-to-import-source-code/importing-an-external-git-repository-using-the-command-line). My main question--does this way migrate all the old merged PRs as well?

**Reply by ajmalmsali** [reply]
Got the same question, how can we import the PRs and related metadata to Github

**Reply by keshavtamang** [reply]
Got the same question, how can we import the PRs and related metadata to Github
